src/test/java/marquez/api/models/JobRequestTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.api.models;
+
+import static java.util.stream.Collectors.toList;
+import static marquez.common.models.CommonModelGenerator.newDescription;
+import static marquez.common.models.CommonModelGenerator.newLocation;
+import static marquez.common.models.Description.NO_DESCRIPTION;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import marquez.UnitTests;
+import marquez.common.models.CommonModelGenerator;
+import marquez.common.models.DatasetUrn;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(UnitTests.class)
+public class JobRequestTest {
+  private static final List<String> INPUT_DATASET_URNS =
+      CommonModelGenerator.newDatasetUrns(4).stream().map(DatasetUrn::getValue).collect(toList());
+  private static final List<String> OUTPUT_DATASET_URNS =
+      CommonModelGenerator.newDatasetUrns(2).stream().map(DatasetUrn::getValue).collect(toList());
```

Comment:
_Minor_: Let's statically import `CommonModelGenerator.newDatasetUrns()` to simplify initialization to:

```
... = newDatasetUrns(4).stream().map(DatasetUrn::getValue).collect(toImmutableList());
```

Reply:
done